### PR TITLE
Fixes to go with changes in main repo

### PR DESCRIPTION
**Pull request by EliAndrewC**
See https://github.com/magfest/ubersystem/pull/1657

### CODE REVIEW

File: barcode/__init__.py [hunk 1]
```diff
@@ -37,5 +30,5 @@ class BarcodePersonalizedBadgeReport(BarcodeReportMixin, uber.reports.Personaliz
     pass
 
 
-uber.reports.printed_badge_report_type = BarcodePrintedBadgeReport
-uber.reports.personalized_badge_report_type = BarcodePersonalizedBadgeReport
+uber.reports.PrintedBadgeReport = BarcodePrintedBadgeReport
+uber.reports.PersonalizedBadgeReport = BarcodePersonalizedBadgeReport
```
Comment (EliAndrewC) [inline barcode/__init__.py]: This is the only "real" fix.  Everything else is just style changes.

**Comment by EliAndrewC** [comment]
Merging this in now.

FWIW, I agree with Vicki and others that this should probably be under https://github.com/magfest instead of https://github.com/rams because while in theory this could be useful to others and thus is a plugin of general interest, that's true of about half of the plugins we've developed.

I'm only mentioning that because normally I would never merge in a change unilaterally to a ``rams`` repo, but my recollection is that people have been treating this repo as a MAGFest repo, so I feel justified in doing so.  Someone should feel free to jump in an correct me if I'm wrong about this.

**Comment by binary1230** [comment]
I think the other way, barcode should still be a part of rams, and other plugins we wrote too like band, hotel, etc, should get moved to rams.

At some point anyway. The naming of repos and github organizations needs some maintenance soon, kitsuta and others were discussing this somewhere else too.

**Comment by EliAndrewC** [comment]
> I think the other way, barcode should still be a part of rams, and other plugins we wrote too like band, hotel, etc, should get moved to rams.

Fair enough.  FWIW, some of those plugins are definitely more suitable than others, e.g. ``bands`` probably contains WAY Too much MAGFest-specific use cases (e.g. talking about our yearly Child's Play Charity Auction), ``mivs`` just isn't something I see existing outside MAGFest period, etc.  ``hotel`` could be a contender for being generic enough to possibly be useful, especially after we make it more generic than only staff rooms.

Regardless, definitely part of a different discussion.

**Pull request merged**